src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as RadixPopover from '@radix-ui/react-popover';
+
+import { PopoverContext } from './Context';
+
+export enum PopoverVariant {
+  Yellow = 'yellow',
+  Red = 'red',
+  White = 'white',
```

Comment:
Instead of color names, I would prefer to stick with something more descriptive of when to use them. The colors may change, then a variant named "Yellow" may not make sense.

We have similar setup for Panel f.ex, where we have Error (red), Info (yellow) and some others. Instead of "White", it could be "Default" or "Neutral" perhaps? Could also check naming with UX, and see what naming they use.